### ISSUE TITLE
Add Zaptec Sense

### ISSUE THREAD

**Issue by philipostli** [PatrickE94/com.zaptec#43]
Requests here: 
- https://community.homey.app/t/app-pro-zaptec-charging/118631/7
- https://community.homey.app/t/zaptec-app-for-homey/74887/135?u=philipostli

**Comment by philipostli** [comment]
@IceHamster Hi. This would be a great addition to the app. I have access to both Zaptec Pro and Go chargers but no Sense. Would you be interested in lending us a device for testing?

**Comment by bobschijvenaars-DS** [comment]
Agree! I would love have this as well. Currently about to get a Zaptec Go2 + Sense.
Is the app already supporting the Go2 BTW?

**Comment by philipostli** [comment]
Hi. We are looking at it at the moment. Not confirmed yet. If the old Go driver is discovering it, it should work already. If not we can add it as a dedicated device.

**Comment by philipostli** [comment]
As per now they do not share any data for te Zaptec Sense, so we can not integrate it. Will check in later if they chose to add it to their open API.
